src/db-api.ts

```diff
@@ -143,9 +153,10 @@ export function execSQL(sql: string): Promise<any> {
 
 export function execSQLWithParams(
   sql: string,
-  params?: Record<string, any>
+  params?: Record<string, any>,
+  options?: { forcePool?: DbPoolName }
```

Comment:
can we add `DbQueryOptions` here for type?

Reply:
Good point. Changed it.